app/src/main/java/zapsolutions/zap/util/Wallet.java

```diff
@@ -694,13 +687,22 @@ public void onNext(ListPeersResponse value) {
                         @Override
                         public void onNext(ConnectPeerResponse value) {
                             ZapLog.debug(LOG_TAG, "Successfully connected to peer, trying to open channel...");
-                            openChannelConnected(nodeUri,amount);
+                            openChannelConnected(nodeUri, amount);
                         }
 
                         @Override
                         public void onError(Throwable t) {
-                            ZapLog.debug(LOG_TAG, "Error opening channel:" + t.getMessage());
-                            broadcastChannelOpenUpdate(nodeUri, false, t.getMessage());
+                            ZapLog.debug(LOG_TAG, "Error connecting to peer:" + t.getMessage());
+
+                            if (t.getMessage().toLowerCase().contains("connectex")) {
+                                broadcastChannelOpenUpdate(nodeUri, ChannelOpenUpdateListener.ERROR_CONNECTION_REFUSED, t.getMessage());
```

Comment:
Are you sure, contains("connectex") is correct? I got a message that connection was refused before, now I get "Could not establish connection to peer node"
The actual error message from LND I get is:
Error connecting to peer:UNKNOWN: dial tcp 203.132.95.10:9735: connect: connection refused